changelog.txt

```diff
@@ -7,6 +7,7 @@
 * Fix - WooCommerce Payments disappeared from WooCommerce Settings if WooCommerce Subscriptions is activated.
 * Add - REST endpoint to capture payments by order ID
 * Add - Payment request button support for checkout and cart blocks.
+* Fix - Risk level is displayed as a "Numeric" value in transactions CSV #1720'
```

Comment:
Interesting that you incldue the issue number here. We usually don't. I don't see a problem but to make sure, could you ask for someone else's opinion about it?
Also, nit: would be nice to add period at the end. I see some other lines do not stop with period though.
Lastly, please also add to readme.txt.

Reply:
Hmm, I just included it so that someone can easily jump to the issue. Of course, we can always find the related issue by other means.

What do you think, @Automattic/gamma is there any harm in including the issue number as well?

Reply:
>Lastly, please also add to readme.txt.
Any reason, why we are maintaining changelog at two places? Just curious.

Reply:
It's one of the requirement by the release script.

Reply:
>Interesting that you incldue the issue number here. We usually don't. I don't see a problem but to make sure

Removed it for now, to maintain consistency but we can continue the discussion.